open-redirect.cs

```diff
@@ -0,0 +1,24 @@
+[HttpPost] 
```

Comment:
usually we have at least 1 flagging and 1 non-flagging case! We tag the flagging case with `ruleid: open-redirect` and the non-flagging as `ok: open-redirect` so our testing mechanisms can catch them.